project/app/api/database.py

```diff
@@ -58,135 +92,158 @@ def close(self):
         self.connection.close()
 
     def fetch_query_records(self, query: str):
-        cursor = self.connection.cursor()
-        cursor.execute(query)
-        result = cursor.fetchall()
+        """This is a custom query, that returns all the records based on customed query"""
+        conn, cursor = self.conn_curs()  # Establishes connection and cursor
+        cursor.execute(query)  # Execute Query
+        result = cursor.fetchall()  # fetches all records, we can limit # of records by put LIMIT statement in query
+        conn.close
         return result
 
     def fetch_all_records(self):
+        #conn, cursor = self.conn_curs()
+        conn = self.connection
         cursor = self.connection.cursor()
-        # 'ID','country','province','district','district_id','sector','sector_id','cell','cell_id','village','village_id','name','project_code','type','stage','sub_stage','individuals_directly_served','span','lat','long','form','case_safe_id','opportunity_id','inc_income','inc_income_rwf','inc_income_usd','bridge_image']
-        query = f"""SELECT "ID", country, province, district,district_id,sector,sector_id,cell,cell_id,village,village_id,name,project_code,type,stage,sub_stage,individuals_directly_served,span,lat,long,form,case_safe_id,opportunity_id,inc_income,inc_income_rwf,inc_income_usd,bridge_image FROM public."Bridges"  """
-        cursor.execute(query)
-        result = cursor.fetchall()
-        columns = ['ID', 'country', 'province', 'district', 'district_id', 'sector', 'sector_id', 'cell', 'cell_id', 'village', 'village_id', 'name', 'project_code', 'type', 'stage',
-                   'sub_stage', 'individuals_directly_served', 'span', 'lat', 'long', 'form', 'case_safe_id', 'opportunity_id', 'inc_income', 'inc_income_rwf', 'inc_income_usd', 'bridge_image']
-        # columns = ['ID', 'country']
-        df = pd.DataFrame(result, columns=columns)
-        df_json = df.to_json(orient='records')
-        parsed = json.loads(df_json)
-        return parsed
-
-    def fetch_query_given_project(self, project_code: int):
-        cursor = self.connection.cursor()
-        query = f"""SELECT * FROM public."Bridges" where project_code={project_code} """
```

Comment:
This query does not return a by given project code, it returns the entire table.